DisplaySettings/DisplaySettings.cpp

```diff
@@ -3650,30 +3650,66 @@ namespace WPEFramework {
         uint32_t DisplaySettings::getSinkAtmosCapability (const JsonObject& parameters, JsonObject& response) 
         {   //sample servicemanager response:
             LOGINFOMETHOD();
-			bool success = true;
-			dsATMOSCapability_t atmosCapability;
+            bool success = true;
+            bool isValidAudioPort =  false;
+            dsATMOSCapability_t atmosCapability;
+            string audioPort = parameters.HasLabel("audioPort") ? parameters["audioPort"].String() : "NULL";
```

Comment:
Pls update DisplaySettings.json schema to reflect that this API now takes parameters - https://github.com/rdkcentral/rdkservices/blob/sprint/22Q4/DisplaySettings/DisplaySettings.json#L952. This will also help update documentation.

Reply:
Also would be good to update versioning (minor version update) and changelog since this is API change - https://github.com/rdkcentral/rdkservices#versioning